### ISSUE TITLE
Feature suggestion: Better offense message for Style/RedundantReturn

### ISSUE THREAD

**Issue by jaredbeck** [bbatsov/rubocop#4157]
## Goal

Provide an offense message that will be more helpful to less experienced ruby programmers.

## Minimal Relevant Config

```
Style/RedundantReturn:
  AllowMultipleReturnValues: false
```

## Actual behavior

```
app/controllers/redacted_controller.rb:264:5: C: Redundant return detected.
    return a, b
    ^^^^^^
```

## Suggested behavior

```
app/controllers/redacted_controller.rb:264:5: C: Redundant return detected. To return multiple values, use an array.
    return a, b
    ^^^^^^
```

## Diff

```diff
- app/controllers/redacted_controller.rb:264:5: C: Redundant return detected.
+ app/controllers/redacted_controller.rb:264:5: C: Redundant return detected. To return multiple values, use an array.
```

## RuboCop version

```
0.47.1 (using Parser 2.4.0.0, running on ruby 2.3.3 x86_64-darwin16)
```

**Comment by gohdaniel15** [comment]
I'm working on it 👍

**Comment by jaredbeck** [comment]
> bbatsov closed this in 5478161 5 hours ago

Thanks, Daniel and Bozhidar.
